doc/deployment/migrations.md

```diff
@@ -22,6 +22,8 @@ Note that the `pachctl migrate` command can be run either before or after you've
 
 Most importantly, you need to ensure that your cluster is "at rest" when you run `pachctl migrate`.  That is, there shouldn't be any ongoing activities that are changing the state of the cluster.  Examples would be running jobs or ongoing `put-file` requests.
 
+*Due to technical reasons, if you have a pipeline that specifies more than one environment variables in its pipeline spec, the first job that the pipeline spawns after the migration will re-process all input.* 
```

Comment:
I actually like @derekchiang original wording better. You could just take the "Due to technical reasons" if you find that to be too tongue in cheek.